sbc_compassion/models/import_letters_history.py

```diff
@@ -244,7 +245,7 @@ def _run_analyze(self):
         self.import_completed = True
         logger.info("Imported letters analysis completed.")
 
-    def _analyze_attachment(self, file_data, file_name):
+    def _analyze_attachment(self, file_data, file_name, is_zipfile=False):
```

Comment:
I know it's my problem, I don't find the way to simplify method redefinition of analyze_attachment in the inherited code (sbc_emai) wihtout add this params.

Without that, redefinition of _run_analyze()(who call analyze_attachment) in sbc_email becomes bigger